apps/spruce/src/utils/index.ts

```diff
@@ -1,7 +1,7 @@
+import * as environmentVariables from "@evg-ui/lib/utils/environmentVariables";
```

Comment:
This is unecessary we can just import from this file in the places this is used.